compiler/src/dotty/tools/io/AbstractFile.scala

```diff
@@ -136,12 +136,6 @@ abstract class AbstractFile extends Iterable[AbstractFile] {
   /** Does this abstract file represent something which can contain classfiles? */
   def isClassContainer: Boolean = isDirectory || (jpath != null && ext.isJarOrZip)
 
-  /** Create a file on disk, if one does not exist already. */
-  def create(): Unit
```

Comment:
We should not delete these now. These might be useful later. Either way, this is not part of the windows performance fix.

Reply:
it is part of pr because it conflicts with cached isDirectory
can't leave it be, waiting to bite someone